contexts/issue.tsx

```diff
@@ -114,9 +114,9 @@ export const IssueProvider: React.FC = function ({ children }) {
     [activeNetwork, query?.repoId, query?.id, user?.accessToken]);
 
   const getNetworkIssue = useCallback(async () => {
-    if (!activeIssue?.contractId || !DAOService?.network?.contractAddress)
+    if ((!activeIssue?.contractId || !DAOService?.network?.contractAddress) || !wallet?.address)
```

Comment:
An user can access web-network from PC and have metamask installed, but don't connected a wallet yet, we still need to load the blockchain data in that case.

I think that for the mobile issue the condition `!DAOService?.network?.contractAddress` is enough, because `contractAddress` on `DAOService.network` will only exists after the contract loads, and this will only happen if metamask is installed.